psychopy/experiment/components/__init__.py

```diff
@@ -157,7 +157,11 @@ def getComponents(folder=None, fetchIcons=True):
         try:
             module = import_module(explicit_rel_path, package=pkg)
         except ImportError:
+            logging.error(
+                'Failed to load component package `{}`. Does it have a '
+                '`__init__.py`.`'.format(cmpfile))
```

Comment:
Really minor edit, but we've got non-matching ` here and no question mark. I think this line should be:
```
'`__init__.py`.?'.format(cmpfile))
```

Reply:
Okay I'll fix it.